.github/format.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+source $(dirname "${BASH_SOURCE[0]}")/bot-pr-comment-base.sh
+
+if [[ "$PR_MERGED" == "true" ]]; then
+  bot_error "PR already merged!"
+fi
+
+git remote set-url origin "$BASE_URL"
+git remote add fork "$HEAD_URL"
+
+# make sure branches are up-to-date
+git fetch origin $BASE_BRANCH
+git fetch fork $HEAD_BRANCH
+
+git config user.email "ginkgo.library@gmail.com"
+git config user.name "ginkgo-bot"
+
+# save scripts from develop
+cd dev_tools/scripts
+cp add_license.sh update_ginkgo_header.sh format_header.sh ../../../
+cd ../../
+
+LOCAL_BRANCH=format-tmp-$HEAD_BRANCH
+git checkout -b $LOCAL_BRANCH fork/$HEAD_BRANCH
+
+# restore files from develop
+cp ../add_license.sh dev_tools/scripts/
+cp ../update_ginkgo_header.sh dev_tools/scripts/
+cp ../format_header.sh dev_tools/scripts/
```

Comment:
This is for security reasons ? So that only the scripts from develop are used ?

Reply:
Exactly, otherwise it would be possible to run code with read/write privileges on the repo if somebody accidentally comments `check-format!` on a malicious PR.